force-app/main/default/lwc/showCustomUiScreenAction/showCustomUiScreenAction.css

```diff
@@ -0,0 +1,539 @@
+/**
+ * 98.css
```

Comment:
We may need to check with legal if we're allowed to use this CSS styles. It might be safe to build our own custom UI.